src/lucky/context_extensions.cr

```diff
@@ -24,4 +24,10 @@ class HTTP::Server::Context
   def flash : Lucky::FlashStore
     @_flash ||= Lucky::FlashStore.from_session(session)
   end
+
+  @_params : Lucky::Params?
+
+  def params : Lucky::Params
```

Comment:
Side bonus, `params` is now available in all Components thanks to https://github.com/luckyframework/lucky/pull/1488